plugins/pulp_rpm/plugins/importers/yum/config_validate.py

```diff
@@ -25,3 +45,41 @@ def validate(config):
             msg += failure_message + '\n'
         msg = msg.rstrip()  # remove the trailing \n
         return False, msg
+
+
+def _validate_allowed_keys(config, error_messages):
+    """
+    Validates that passed signature keys have proper length.
+
+    :param config: configuration instance passed to the importer
+    :type  config: pulp.plugins.config.PluginCallConfiguration
+
+    :param error_messages: errors encountered during validation
+    :type error_messages: list
+    """
+
+    allowed_keys = config.get('allowed_keys', [])
+    for key in allowed_keys:
+        if len(key) != 8:
+            msg = _('Signature key %s should be 8 characters long') % key
+            error_messages.append(msg)
+
+
+def _validate_lazy_compatibility(config, error_messages):
+    """
+    Validates lazy sync and signature policy compatibility.
+    To extract the signature information from the package, we need this package been downloaded.
+
+    :param config: configuration instance passed to the importer
+    :type  config: pulp.plugins.config.PluginCallConfiguration
+
+    :param error_messages: errors encountered during validation
```

Comment:
It would be helpful to state the side effect that this function may add items to the list, and what the circumstances would be to cause that.